internal/tool/ssh/impl/manager.go

```diff
@@ -313,15 +312,12 @@ func (m *manager) Status(ctx context.Context) (*status.Status, error) {
 	var components []*status.Component
 	for _, g := range m.allGroups() {
 		g.components.Lock()
-		cs := maps.Keys(g.components.Val)
-		g.components.Unlock()
-		g.mu.Lock()
```

Comment:
let's remove just this:
`
g.mu.Lock()
pids := slices.Clone(g.pids)
g.mu.Unlock()
`

Reply:
Yes fixed it now, though I was doing something wrong the last time :)